### PR TITLE
Smallpatches

### PR DESCRIPTION
**Pull request by InfusOnWoW**
Maybe, we should call SetDurationInfo with nil, nil instead.

But that would require further changes down the line

Ticket-Nr: 1010

### CODE REVIEW

**Comment by InfusOnWoW** [comment]
The Item Set change could use a bit more work.

File: WeakAuras/Prototypes.lua [hunk 1]
```diff
@@ -3404,7 +3419,45 @@ WeakAuras.event_prototypes = {
...
+      }
+    },
+    durationFunc = function(trigger)
+      return WeakAuras.GetNumSetItemsEquipped(trigger.itemSetId and tonumber(trigger.itemSetId) or 0)
+    end,
+    nameFunc = function(trigger)
+      return select(3, WeakAuras.GetNumSetItemsEquipped(trigger.itemSetId and tonumber(trigger.itemSetId) or 0));
+    end
+  },
+  ["Item Set Equipped"] = { -- Equipment Set
```
Comment (Stanzilla) [inline WeakAuras/Prototypes.lua]: Should probably rename that to Equipment Set Equipped to not be confusing.

Comment (Stanzilla) [inline WeakAuras/Prototypes.lua]: AceGUIWidget-WeakAurasDisplayButton.lua needs the minor bumped at the top

File: WeakAuras/Prototypes.lua [hunk 2]
```diff
@@ -3404,7 +3419,63 @@ WeakAuras.event_prototypes = {
...
+
+          local sets = C_LootJournal.GetFilteredItemSets();
+          C_LootJournal.SetClassAndSpecFilters(classFilter, specFilter);
+
+          local description = "";
+          for index, set in ipairs(sets) do
+            description = description .. set.name .. ": " .. set.setID .. "\n";
+          end
+
+          description = description .. "\n" .. L["Older sets ids can be find on websites such as wowhead.com"];
```
Comment (Stanzilla) [inline WeakAuras/Prototypes.lua]: "Older set IDs can be found on websites such as wowhead.com/item-sets"

File: WeakAuras/Prototypes.lua [hunk 3]
```diff
@@ -3404,7 +3419,63 @@ WeakAuras.event_prototypes = {
...
+    name = L["Item Set Equipped"],
+    automaticrequired = true,
+    init = function(trigger)
+      return string.format("local setid = %s;\n", trigger.itemSetId and tonumber(trigger.itemSetId) or "0");
+    end,
+    statesParameter = "one",
+    args = {
+      {
+        name = "itemSetId",
+        display = L["Item Set"],
```
Comment (Stanzilla) [inline WeakAuras/Prototypes.lua]: should probably add "id" here as well so it's clear what the input field wants

File: WeakAuras/Transmission.lua [hunk 4]
```diff
@@ -826,6 +827,11 @@ function WeakAuras.ShowDisplayTooltip(data, children, icon, icons, import, compr
       tinsert(tooltip, {1, data.url, 1, 0.82, 0, 1});
     end
 
+    if (hasVersion) then
+      tinsert(tooltip, {1, L["Version: "] .. data.version, 1, 0.82, 0, 1});
+    end
+
+
```
Comment (Stanzilla) [inline WeakAuras/Transmission.lua]: YOU AND YOUR BLANK LINES

**Pull request merged**